### ISSUE TITLE
[HOLD for payment 2024-08-13] [Fullstory] Unmasking session data in Fullstory

### ISSUE THREAD

**Issue by anmurali** [Expensify/App#46579]
**Problem**: We have the FS implementation done 95% of the way. There is still a bit of clean up to do and we're working on it but the integration is ready for us to use. There are a multitude of ways to use this data but a pretty core one is watching session data to understand pockets of user confusion. But right now we're sending the data completely masked. When we observe the session, it is a hard to follow what the user is even clicking on. We tried selectively unmasking the data but this is a piecemeal effort across many many pages and is effectively going to delay us getting value out of this tool in short order

**Solution**: Lets unmask the data by default and get started. Our user segments exclude emails with domain `expensify.com` so by design we won't be tracking or watching those sessions. And all of us have access to supportal anyway, where all the data is available. 

Lean Convert will be working on this build
[Expensify Order Form Experimentation 18-07-24.docx (2) (1).pdf](https://github.com/user-attachments/files/16442888/Expensify.Order.Form.Experimentation.18-07-24.docx.2.1.pdf)

**Comment by luacmartins** [comment]
It'd be great if @rushatgabhane can take investigation/QA over.

**Comment by rushatgabhane** [comment]
sounds good

**Comment by anmurali** [comment]
FS is still looking into this. No updates yet.

**Comment by sakluger** [comment]
@luacmartins can you please confirm who needs to be paid for this issue? It's hard to tell based on the issue assignees vs the PR author and reviewer.

**Comment by luacmartins** [comment]
That would be @LCOleksii for the 1st PR linked to the issue. I'm not sure if they have some pre-agreement for this project though. @anmurali can you confirm?

**Comment by sakluger** [comment]
@LCOleksii how are you getting paid for this PR? Through Upwork, NewDot, or is it already handled elsewhere?

**Comment by sakluger** [comment]
I asked for guidance around payments on Slack: https://expensify.slack.com/archives/C02NK2DQWUX/p1723826353151379

**Comment by sakluger** [comment]
@luacmartins do you @LCOleksii's email or name on Slack? @anmurali is OOO for the rest of the week and @LCOleksii has not responded on this GH issue, so I'm not sure how else to find out what needs to happen for payments.

**Comment by sakluger** [comment]
Oh - I see an order form with LeanConvert in the original comment/description for this GH issue, which confirms that we're paying them separately.

I do see that @rushatgabhane reviewed [the PR](https://github.com/Expensify/App/pull/47401), so I think we do actually have to pay Rushat. @luacmartins can you please confirm if Rushat is due payment for reviewing?

**Comment by luacmartins** [comment]
I think so, but maybe @rushatgabhane is also being paid a fixed price for being a dedicated C+ to this project? @rushatgabhane can you please confirm?

**Comment by rushatgabhane** [comment]
Hey @luacmartins the PR did not require any time to review because we just added a new class name. So no action required here

**Comment by sakluger** [comment]
Got it, thanks for clarifying!  I assume that @LCOleksii was paid in a separate agreement, so I will close for now. Feel free to reopen if additional payment is required.
